src/entities/direct-thread.entity.ts

```diff
@@ -1,12 +1,21 @@
 import * as urlRegex from 'url-regex';
 import { Entity } from '../core/entity';
-import { DirectThreadBroadcastPhotoOptions } from '../types/direct-thread.broadcast-photo.options';
-import { DirectThreadBroadcastOptions } from '../types/direct-thread.broadcast.options';
+import { DirectThreadBroadcastPhotoOptions } from '../types';
+import { DirectThreadBroadcastOptions } from '../types';
+import { IgClientError } from '../errors';
 
 export class DirectThreadEntity extends Entity {
   threadId: string = null;
   userIds: string[] = null;
 
+  public async deleteItem(itemId: string | number, threadId?: string | number) {
```

Comment:
Why do we need `threadId` param here? If we work with concrete direct thread entity we shouldn't be able to delete item from another direct thread.

Reply:
The threadId didn't set properly using the Feed, so I've just added it. But now I hope it's fixed.